main/acle.md

```diff
@@ -1509,24 +1509,32 @@ SVE language extensions:
 > [The __ARM_FEATURE_SVE_BITS macro](#the-__arm_feature_sve_bits-macro)
 > for details.
 
-**`__ARM_FEATURE_SVE_VECTOR_OPERATORS==1`**
+**`__ARM_FEATURE_SVE_VECTOR_OPERATORS==N`**
 
-> This indicates that applying the `arm_sve_vector_bits` attribute
+> When N is 1, this indicates that applying the `arm_sve_vector_bits` attribute
 > to an SVE vector type creates a type that supports the GNU vector
-> extensions. The state of this macro is only meaningful when
+> extensions. This value of this macro is only meaningful when
 > `__ARM_FEATURE_SVE_BITS` is nonzero. See [`arm_sve_vector_bits` behavior
 > specific to vectors](#arm_sve_vector_bits-behavior-specific-to-vectors)
 > for details.
 
+> When N is 2, this indicates that the operators outlined in the GNU vector
+> extensions additionally work on all SVE vector types. This value of this
+> macro still has meaning when `__ARM_FEATURE_SVE_BITS` is nonzero.
```

Comment:
I wonder whether we should instead say that the meaning/condition for sizeless types is independent of `__ARM_FEATURE_SVE_BITS`.  The fact that it applies even without `__ARM_FEATURE_SVE_BITS` is arguably more surprising :-)